src/authorization/OwnerPermissionReader.ts

```diff
@@ -17,40 +22,54 @@ export class OwnerPermissionReader extends PermissionReader {
 
   private readonly accountStore: AccountStore;
   private readonly aclStrategy: AuxiliaryIdentifierStrategy;
+  private readonly identifierStrategy: IdentifierStrategy;
 
-  public constructor(accountStore: AccountStore, aclStrategy: AuxiliaryIdentifierStrategy) {
+  public constructor(accountStore: AccountStore, aclStrategy: AuxiliaryIdentifierStrategy,
+    identifierStrategy: IdentifierStrategy) {
     super();
     this.accountStore = accountStore;
     this.aclStrategy = aclStrategy;
+    this.identifierStrategy = identifierStrategy;
   }
 
-  public async handle(input: PermissionReaderInput): Promise<PermissionSet> {
+  public async handle(input: PermissionReaderInput): Promise<PermissionMap> {
+    const result: PermissionMap = new IdentifierMap();
+    const acls = [ ...filter(input.requestedModes.distinctKeys(),
+      (id): boolean => this.aclStrategy.isAuxiliaryIdentifier(id)) ];
```

Comment:
```suggestion
    const requestedResources = input.requestedModes.distinctKeys();
    const acls = [ ...filter(requestedResources, (id): boolean => this.aclStrategy.isAuxiliaryIdentifier(id)) ];
```
(I spend too long wondering about the original first line.)